common/src/main/java/com/wynntils/core/features/AbstractConfigurable.java

```diff
@@ -31,7 +31,7 @@ public final List<ConfigHolder> getConfigOptions() {
 
     @Override
     public final Optional<ConfigHolder> getConfigOptionFromString(String name) {
-        return getVisibleConfigOptions().stream()
+        return getConfigOptions().stream()
```

Comment:
Features did visible config options, overlays did all config options. Everywhere this is used, we need to filter from all config options.